apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_txt2img.py

```diff
@@ -39,8 +39,9 @@ def __init__(
             SharkEulerDiscreteScheduler,
             DEISMultistepScheduler,
         ],
+        lora_path: str = "",
     ):
-        super().__init__(vae, text_encoder, tokenizer, unet, scheduler)
+        super().__init__(vae, text_encoder, tokenizer, unet, scheduler, lora_path)
```

Comment:
I've a couple of questions here for my understanding :-

I just checked the super class `StableDiffusionPipeline` and it has no `lora_path` argument as part of its `__init__` or even a member variable expecting one.

So, why do we require this change in `txt2img` pipeline?

Also, I believe this change would bail out at the interpreter/compiler in the first place. Was this tested?